### ISSUE TITLE
EmptyLineSeparator fails on multiple imports or fields

### ISSUE THREAD

**Issue by mkordas** [checkstyle/checkstyle#706]
##### Description
[EmptyLineSeparator](http://checkstyle.sourceforge.net/config_whitespace.html#EmptyLineSeparator) check with new option `allowMultipleEmptyLines` set to `false` fails when there is  more than 1 import in file or there are consecutive fields in file and `allowNoEmptyLineBetweenFields` is set to `true`.

##### Configuration used
Checkstyle 6.4 and the following check:
```xml
<module name="EmptyLineSeparator">
  <property name="allowMultipleEmptyLines" value="false"/>
  <property name="allowNoEmptyLineBetweenFields" value="true"/>
</module>
```

##### Input file
```java
package com.puppycrawl.tools.checkstyle.whitespace;

import java.util.Calendar;
import java.util.Date; // 4: 'import' has more than 1 empty lines before

public class InputEmptyLineSeparatorCheckMultipleImportsAndFields
{
    Date date;
    Calendar calendar; // 9: 'VARIABLE_DEF' should be separated from previous statement
}
```
##### Expected
No violations reported for the input file.

**Comment by romani** [comment]
merged, will be in 6.5
